index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+
+  <script>
+    function freeCoffeeCups(coffeeCups) {
+      var add = 0;
+      if(coffeeCups === 1) {
```

Comment:
Why have you isolated this particular condition? Based on your code for your else, it looks like what you wrote in your else statement would work for 1 as well 🤔